s/starlark-go/starlark-go_ubi_8.5.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : starlark-go
+# Version       : d1966c6
+# Source repo   : https://github.com/google/starlark-go
+# Tested on     : UBI 8.5
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vathsala .<vaths367@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=starlark-go
+PACKAGE_VERSION=${1:-d1966c6}
+PACKAGE_URL=https://github.com/google/starlark-go
+
+SCRIPT=$(readlink -f $0)
+SCRIPT_DIR=$(dirname $SCRIPT)
+PATCH_FILE=$SCRIPT_DIR/starlark.patch
+
+yum install -y git  make gcc diffutils wget patch
+
+wget https://golang.org/dl/go1.17.linux-ppc64le.tar.gz
+rm -rf /home/go && tar -C /home -xzf go1.17.linux-ppc64le.tar.gz
+rm -f go1.17.linux-ppc64le.tar.gz
+export GOPATH=/home/go
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+export  GO111MODULE=on
+
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
```

Comment:
You may want to add a check for the git clone command.

Reply:
@vathsalashetty25 Any update on this?

Reply:
Yes..Instead of Patch file,i have used sed command...it's working same as patchfile

Reply:
yes..i have added check for git clone command